docker/scripts/build-h2o-3

```diff
@@ -8,7 +8,7 @@ if [[ ! ${H2O_BRANCH} ]]; then
 fi
 if [[ ! -n ${PYTHON_VERSION} ]]; then
   echo "Using default Python version"
-  PYTHON_VERSION='3.6'
+  PYTHON_VERSION='3.7'
```

Comment:
Python 3.6 is not supported anymore by the ubuntu 22.04 and is installed (built from source) separately for the py-3.6 tests.